lib/sedna/__init__.py

```diff
@@ -13,3 +13,4 @@
 # limitations under the License.
 
 from .__version__ import __version__
+from sedna.algorithms import *
```

Comment:
Please avoid import *, cause it would lead name conflicts.

Reply:
To ensure that algorithms are registered in the memory, all algorithms need to be traversed when sedna is import.